package-lock.json

```diff
@@ -9620,11 +9620,6 @@
         }
       }
     },
-    "load-script": {
```

Comment:
I'm a bit confused as to how these have been removed, and I'd probably prefer not to change the package-lock.json where we don't need to (nothing in this PR affects the packages that have been removed)

Reply:
On second thought don't worry about thsi you might just need to merge main in